src/main/groovy/com/github/eirnym/js2p/JsonSchemaExtension.groovy

```diff
@@ -38,7 +40,7 @@ import javax.inject.Inject
  * @see https://github.com/joelittlejohn/jsonschema2pojo
  */
 public class JsonSchemaExtension implements GenerationConfig {
-  Iterable<File> sourceFiles
+  final ConfigurableFileCollection sourceFiles
```

Comment:
Here we change the type of field, but backward compatibility will be kept by the `setSource(Iterable<File> files)` method.

Reply:
make sense, my plugin was just a merge between two others